docs/deploy/redmine.rst

```diff
@@ -0,0 +1,337 @@
+Redmine tasks
+=============
+
+The `Data Support CRM <https://crm.open-contracting.org>`__ uses `Redmine <https://www.redmine.org>`__.
+
+Enable Admin user
```

Comment:
@jpmckinney , Just added this section regarding enabling the admin user, not 100% sure if this is the right page for it? perhaps better under the maintenance section.

Reply:
We can leave it here for now, until we have more maintenance tasks.